src/functionalTests/resources/json/bulkscan/basic.json

```diff
@@ -0,0 +1,9 @@
+{
+  "case_type_id": "FinancialRemedyContested",
+  "id": "LV481297",
+  "po_box": "PO 17",
+  "form_type": "form_a",
+  "ocr_data_fields": [
+    { "name": "field", "value": "content" }
```

Comment:
Validation endpoint will fail with these values I guess - but maybe that's not an issue for this particular task.